tests/Unit/Domain/Test_LogicalCoordinates.cpp

```diff
@@ -15,10 +15,11 @@
 #include "Domain/CoordinateMaps/CoordinateMap.tpp"
```

Comment:
Rename file to `Test_InterfaceLogicalCoordinates.cpp` and move the logical_coordinates test to the other directory